src/presign/participant.rs

```diff
@@ -1267,7 +1267,10 @@ mod test {
             .map(|record| record.mask_share())
             .fold(Scalar::ZERO, |sum, mask_share| sum + mask_share);
         let inverse: Scalar = Option::from(mask.invert()).unwrap();
-        assert_eq!(mask_point.0, k256::ProjectivePoint::GENERATOR * inverse);
+        assert_eq!(
+            mask_point.get_zero(),
+            k256::ProjectivePoint::GENERATOR * inverse
```

Comment:
todo: the second value is a scalar multiplication, so I think it's better to use the `CurvePoint::generator` and `multiply_by_scalar` methods than to require access to the k256 types. Then we can compare directly to `mask_point`.